pkg/jsonpath/v1alpha1/json_path.go

```diff
@@ -33,6 +34,7 @@ type selection struct {
 	*Msgs
```

Comment:
And then you need to put.
`*pkg_msg_v1alpha1.Msg`
And at all other places where it would expect.
And run `make test` to validate changes